### PR TITLE
Minor public updates

### PR DESCRIPTION
**Pull request by olaaustine**
[ENSVAR-4680](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-4680)
Information : Pull request to update the public docs on the ensembl website

### CODE REVIEW

**Comment by ens-ma7** [comment]
Hi @olaaustine any updates on this PR? Is it awaiting approval? can you get someone to review it and approve please.

File: docs/htdocs/info/docs/tools/vep/script/vep_cache.html [hunk 1]
```diff
@@ -102,7 +102,7 @@ <h3 id="manual_cache">Manually downloading caches</h3>
 
   <ul>
     <li>
-      <p><b>Indexed cache</b> (<a href="[[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/indexed_vep_cache/">[[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/indexed_vep_cache/</a>) - requires <a href="https://github.com/Ensembl/Bio-DB-HTS" rel="external">Bio::DB::HTS</a> (setup by INSTALL.pl) or <a href="https://github.com/samtools/tabix" rel="external">tabix</a>, e.g.:</p>
+      <p><b>Indexed cache</b> <p>essential for human and other species with large sets of variant data</p> (<a href="[[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/indexed_vep_cache/">[[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/indexed_vep_cache/</a>) - requires <a href="https://github.com/Ensembl/Bio-DB-HTS" rel="external">Bio::DB::HTS</a> (setup by INSTALL.pl) or <a href="https://github.com/samtools/tabix" rel="external">tabix</a>, e.g.:</p>
```
Comment (diegomscoelho) [inline docs/htdocs/info/docs/tools/vep/script/vep_cache.html:105]:
```suggestion
      <p><b>Indexed cache</b> <p>Essential for human and other species with large sets of variant data</p> (<a href="[[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/indexed_vep_cache/">[[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/indexed_vep_cache/</a>) - requires <a href="https://github.com/Ensembl/Bio-DB-HTS" rel="external">Bio::DB::HTS</a> (setup by INSTALL.pl) or <a href="https://github.com/samtools/tabix" rel="external">tabix</a>, e.g.:</p>
```

**Review by diegomscoelho — COMMENTED** [review]
Just this minor style update.

**Review by diegomscoelho — APPROVED** [review]
LOTM! Good to go @ens-ma7!

**Comment by olaaustine** [comment]
Added the suggestion. Thanks @diegomscoelho

**Pull request merged**